### DISCUSSION TITLE
Stream in a Vercel Serverless function by Python (Flask) only  working in local, but not once deployed

### DISCUSSION THREAD

**Discussion by zhuchaokn** [vercel/community#2473] [Help]
### Summary

A simple Case  of streaming response ,code like this 👍 
```
from flask import Flask, Response
import time

app = Flask(__name__)

def generate_text():
    for i in range(10):
        yield "This is line %d.\n" % i
        time.sleep(1)

@app.route('/')
def stream_text():
    return Response(generate_text(), mimetype='text/plain')

if __name__ == '__main__':
    app.run(debug=True)
```
It worked local, bu not once deployed, Always wating full response and met "INVACATION TIME OUT" finally. 
So how can I make a correct stream response in Flask.

**Comment by zhuchaokn** [comment]
I may have found the reason. Because the HTTP request is actually forwarded through Nginx or other proxies, when configuring the forwarding rules, the following settings are required: 
```
proxy_http_version 1.1;  
proxy_set_header Connection "" ;
```
After adding these two configuration lines to Nginx, it became effective.

**Reply by smadikanti** [reply]
How are these settings applicable to Vercel env? Unsure on how/where to configure these values for a vercel env.
